src/tests/base_test.py

```diff
@@ -153,6 +207,42 @@ def new_cluster_configuration(self, request: FixtureRequest) -> ClusterConfig:
 
         return config
 
+    @pytest.fixture
+    def new_day2_cluster_configuration(
```

Comment:
(a general comment, not specific to this line)
make sure to check if this PR has any impact on qe (or maybe you already checked)

Reply:
yes, once I get this PR working, I'll ping QE